docs/src/examples/axislike.md

```diff
@@ -110,3 +110,19 @@ savefigs("polar", ans) # hide
 [\[.pdf\]](polar.pdf), [\[generated .tex\]](polar.tex)
 
 ![](polar.svg)
+
+## Smith Chart
+
+```@example pgf
+frequency = Array(range(100e6,stop=10e9,length=10)) # 100 MHz to 10 GHz
+L = 1e-9 # 1 nH
+R = 25   # 25 Ω
+Z0 = 50  # 50 Ω Reference
+network = (R .+ 1.0im*2*pi*frequency*L) ./ Z0 # Series network of R + jωL, normalized
```

Comment:
Also, perhaps

```
network = @. R + 1.0im*2*pi*frequency*L) / Z0
```

Reply:
And the `1.0` before `im` should be redundant?

Reply:
I think the 1.0 is good for clarity. The Julia docs seem to include constants in all their expressions involving `im`

Reply:
But I can remove it if you'd like @KristofferC

Reply:
Doesn't matter much to me, although I would question if adding a redundant multiplication by 1 increases clarity. But if the rest of the julia docs does it, lets keep it.